api/src/cli/commands/security/key.ts

```diff
@@ -0,0 +1,8 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export default async function generateKey(): Promise<void> {
+	const key = uuidv4();
+
+	process.stdout.write(`Your new app key: ${key}\n`);
```

Comment:
Lets return the value immediately, instead of prefixing it with the message. That'll allow CI to use it as well (where you can pipe the output into something else)

Reply:
I totally forgot about this PR haha. Should we still return a new line after the value?

Reply:
> Should we still return a new line after the value?

Nah, lets just return the raw value and nothing else. That way it's easiest for external processes to use the generated information 🙂

Reply:
Changed!

Reply:
Yup!